### ISSUE TITLE
opmerkingen week 1

### ISSUE THREAD

**Issue by bart314** [MartijnRozema/ML_2023-P1#1]
In de [intro-notebook](Deel-1/intro notebook en sklearn.ipynb) schrijven jullie wel de code correct, maar wordt verzuimd om de reflectie-vragen te beantwoorden. Zoals waarom je welke hoeveelheid bins prefereert of waarom de split nodig is. Ook geven jullie geen mogelijkheden waarmee je het model zou kunnen verbeteren.

De uitwerkingen van de gradient descent zijn prima, maar waar komt dat `RemoveElement` (vanaf r.140) vandaan 🤔?
